rockcraft/extensions/gunicorn.py

```diff
@@ -66,13 +66,20 @@ def gen_install_app_part(self) -> Dict[str, Any]:
     def _gen_parts(self) -> dict:
         """Generate the parts associated with this extension."""
         data_dir = get_extensions_data_dir()
+        stage_packages = ["python3-venv"]
+        build_environment = []
+        if self.yaml_data["base"] == "bare":
+            stage_packages = ["python3.10-venv_ensurepip"]
+            build_environment = [{"PARTS_PYTHON_INTERPRETER": "python3.10"}]
```

Comment:
is there no slice that provides the `python3` symlink? (just out of curiosity)

Reply:
Python3 is available in the final image, but during this phase python3-venv apt package is automatically installed by the python plugin, and it gets in conflict with the slice I think. Couldn't make it work without the build-environment